### PR TITLE
autocomplete attributes added

### PR DESCRIPTION
**Pull request by Pankaj-Wadhwani**
Signed-off-by: Pankaj Wadhwani <pankajwadhwani100@gmail.com>

### Description

Added `autocomplete : on` in form tag,username field and password field.

Ref #16001

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by MauricioFauth — COMMENTED** [review]
Is there a reason to add autocomplete for username and password fields?

**Comment by mauriciofauth** [comment]
Okay. It's related to #16001.

**Comment by williamdes** [comment]
From what I linked in #16001 https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands the value could be more precise then `on`

**Comment by Pankaj-Wadhwani** [comment]
what if i do like this,
form tag has autocomplete="off"
username has autocomplete="pma_username"
password has autocomplete="pma_password"

**Comment by williamdes** [comment]
> what if i do like this,
> form tag has autocomplete="off"
> username has autocomplete="pma_username"
> password has autocomplete="pma_password"
> @williamdes

I am not sure this is standard, could you do some research ?

**Comment by Pankaj-Wadhwani** [comment]
> 
> I am not sure this is standard, could you do some research ?

yes

**Comment by Pankaj-Wadhwani** [comment]
I changed the commit in pull request,please review it. @williamdes

File: templates/login/form.twig [hunk 1]
```diff
@@ -46,7 +46,7 @@
         {% trans 'Username:' %}
       </label>
       <div class="col-8">
-        <input type="text" name="pma_username" id="input_username" value="{{ default_user }}" size="24" class="textfield form-control">
+        <input type="text" name="pma_username" id="input_username" value="{{ default_user }}" size="24" class="textfield form-control" autocomplete="pma_username">
```
Comment (mauriciofauth) [inline templates/login/form.twig]: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

```suggestion
        <input type="text" name="pma_username" id="input_username" value="{{ default_user }}" size="24" class="textfield form-control" autocomplete="username">
```

File: templates/login/form.twig [hunk 2]
```diff
@@ -55,7 +55,7 @@
         {% trans 'Password:' %}
       </label>
       <div class="col-8">
-        <input type="password" name="pma_password" id="input_password" value="" size="24" class="textfield form-control">
+        <input type="password" name="pma_password" id="input_password" value="" size="24" class="textfield form-control" autocomplete="pma_password">
```
Comment (mauriciofauth) [inline templates/login/form.twig]: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

```suggestion
        <input type="password" name="pma_password" id="input_password" value="" size="24" class="textfield form-control" autocomplete="current-password">
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by Pankaj-Wadhwani** [comment]
Review my commit @mauriciofauth

**Review by MauricioFauth — APPROVED** [review]

**Comment by williamdes** [comment]
Thank you @Pankaj-Wadhwani for your contribution!

**Comment by Pankaj-Wadhwani** [comment]
> Thank you @Pankaj-Wadhwani for your contribution!

My pleasure.
Would love to contribute more.

**Pull request merged**